
#include "centre_player_response_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void CentrePlayerUtilityReply::SendTipToClient(entt::entity player,const ::TipInfoMessage* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerUtilityReply::KickPlayer(entt::entity player,const ::CentreKickPlayerRequest* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}

